gitkraken-client/open-clone-init.md

```diff
@@ -56,7 +71,7 @@ This will then prompt you to open the newly copied project in GitKraken Client.
 ### Initialize a new project
 Starting a project in GitKraken Client is easy through <kbd><strong>File > Init</strong></kbd>
 
-<img src='/wp-content/uploads/init.png' srcset='/wp-content/uploads/init@2x.png 2x' class='img-bordered img-responsive center'>
+<img src='wp-content/uploads/gkc-repo-mngmt-init-repo.png' class='img-bordered img-responsive center'>
```

Comment:
Do you know if images work without the leading backslash or should this (and a few others) be _src=`/wp-content/..._